docs/asciidoc/modules/ROOT/pages/database-integration/vectordb/index.adoc

```diff
@@ -0,0 +1,94 @@
+[[vectordb]]
+= Vector Databases
+:description: This section describes procedures that can be used to interact with Vector Databases.
+
+APOC provides these set of procedures, which leverages the Rest APIs, to interact with Vector Databases:
+
+- `apoc.vectordb.qdrant.*` (to interact with https://qdrant.tech/documentation/overview/[Qdrant])
+- `apoc.vectordb.chroma.*` (to interact with https://docs.trychroma.com/getting-started[Chroma])
+- `apoc.vectordb.weaviate.*` (to interact with https://weaviate.io/developers/weaviate[Weaviate])
+- `apoc.vectordb.custom.*` (to interact with other vector databases)
+- `apoc.vectordb.store` (to store host, credentials and mapping into the system database)
+
+All the procedures, except the `apoc.vectordb.store` one, can have, as a final parameter,
+a configuration map with these possible parameters:
+
+.config parameters
+
+|===
+| key | description
+| headers | additional HTTP headers
+| method | HTTP method
+| endpoint | endpoint key, 
+    can be used to override the default endpoint created via the 1st parameter of the `apoc.vectordb.qdrant.*` and `apoc.vectordb.qdrant.*`,
+    to handle potential endpoint changes.
+| body | body HTTP request
+| jsonPath | To customize https://github.com/json-path/JsonPath[JSONPath] of the response. The default is `null`.
+|===
+
+
+Besides the above config, the `apoc.vectordb.<type>.get` and the `apoc.vectordb.<type>.query` procedures can have these additional parameters:
+
+.embeddingConfig parameters
+
+|===
+| key | description
+| mapping | to auto-create entities. See examples below.
+| allResults | if true, returns the vector, metadata and text (if present), otherwise returns null values for those columns.
+| vectorKey, metadataKey, scoreKey, textKey | used with the `apoc.vectordb.custom.get` procedure.
+    To let the procedure know which key in the restAPI (if present) corresponds to the one that should be populated as respectively the vector/metadata/score/text result.
+    Defaults are "vector", "metadata", "score", "text".
+    See examples below.
+|===
+
+include::./qdrand.adoc[]
+
+include::./chroma.adoc[]
+
+include::./weaviate.adoc[]
+
+include::./custom.adoc[]
+
+== Store Vector db info (i.e. `apoc.vectordb.store`) 
+
+We can save some info in the System Database to be reused later, that is the host, login credentials, and mapping,
+to be used in `*.get` and `.*query` procedures, except for the `apoc.vectordb.custom.get` one.
+
+Therefore, to store the vector info, we can execute the `CALL apoc.vectordb.store(vectorName, host, credentialsValue, mapping)`,
```

Comment:
shouldn't I be able to use multiple vector names for the same provider?
like "qdrant" + "books" or "qdrant" + "papers" ? we should not limit it to one only.
so that instead of host+key I would use "books" or "papers"